src/main/java/com/example/everkan/security/jwt/JwtSecurityConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.example.everkan.security.jwt;
+
+public class JwtSecurityConstants {
+    public static final String SECRET = "SecretKeyToGenJWTs";
+    public static final long EXPIRATION_TIME = 86_400_000;
```

Comment:
What's the unit of the expiration time?

You could also store it as a `Duration`, which can be easily converted into all sorts of time units (seconds, milliseconds, ...).

Reply:
The unit of `EXPIRATION_TIME` is milliseconds.  I will adopt your suggestion.